tests/test_mro_calculation.py

```diff
@@ -16,53 +16,3 @@ def mypy_cache_dir(tmp_path_factory):
     tdir = tmp_path_factory.mktemp('.mypy_cahe')
     print("Setup cache", str(tdir))
     return str(tdir)
-
```

Comment:
You can probably delete the entire file now.

Reply:
Oh, indeed!